### ISSUE TITLE
Cannot connect Botium to an SOE which acts as Watson Assistant proxy

### ISSUE THREAD

**Issue by andrewrfreed** [codeforequity-at/botium-core#689]
**Describe the bug**
The WA botium connector is not working with a classic SOE
A customer uses an SOE that proxies their Watson Assistant v1 instance and it expects to be called with Basic authentication.
(`Authorization: Basic base_64_encoded_apikey_colon_actualapikey`)
The SOE application wants to manage the IAM token relationship on its own, not have Botium manage it.
This seems like it should be possible in Botium if a different authenticator is used.  The SOE itself is difficult to update.

I've been unable to test but will submit what code I expect could work.

**To Reproduce**
Configure WA botium connect _without_ using the apikey.   .env file should have BOTIUM_WATSON_USER and BOTIUM_WATSON_PASSWORD.

**Expected behavior**
Botium should connect to the SOE which proxies Watson Assistant, configured only with BOTIUM_WATSON_USER=apikey and BOTIUM_WATSON_PASSWORD=the_actual_api_key

**Screenshots and Log files**
<img width="1345" alt="botium-wa-connector" src="https://user-images.githubusercontent.com/16532202/177870972-8da4b870-43cc-4aaa-b962-b653cb29d8b1.png">

**Botium Flavour:**
 - Botium Box

**Botium Version**
 - Latest

**Additional context**

**Comment by codeforequity-at** [comment]
Right, this is a thing that wasn't considered in the Botium connector yet.
